torch/nn/modules/module.py

```diff
@@ -1572,10 +1582,16 @@ def _call_impl(self, *args, **kwargs):
 
         return result
 
-    __call__ : Callable[..., Any] = _call_impl
+    __call__ : Callable[..., Any] = _wrapped_call_impl
```

Comment:
This is not sound to do carte_blanche, at all. 

Not only does this break inlining, it presumes the presence of _compiled_call_impl - which is also not guaranteed, unless one calls .compile.

Always calling .compile is not sound, or safe, even on a PT2 supported model, because it makes assumptions around backends and config.